src/store/keyboard/context.js

```diff
@@ -0,0 +1,48 @@
+import { createContext, useContext, useState } from 'react';
+
+const KeyboardContext = createContext();
+
+export const KeyboardProvider = ({ children }) => {
+  const [isOptionKeyPressed, setIsOptionKeyPressed] = useState(false);
+
+  const initialvalue = {
+    isOptionKeyPressed,
+  };
+
+  function isOptionKey(e) {
+    if (e.key === 'Alt') {
+      return true;
+    }
+
+    return false;
+  }
```

Comment:
This should be defined outside KeyboardProvider component function(in this file only before line 5)
Otherwise this function will be defined every time the component is rendered.

Reply:
done in the latest commit